README.md

```diff
@@ -86,6 +87,8 @@ public class MyConfig implements WebMvcConfigurer {
 }
 ```
 
+If you need to use additional converters please see [how to configure specification argument resolver with custom converters](#conversions)
```

Comment:
The library converts HTTP parameters (strings) into most popular Java types such as dates, numbers, and enums. In case of a need for some additional conversion, please see the [how to configure specification argument resolver with custom converters](#conversions) section.